src/index.html

```diff
@@ -1,14 +1,44 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body id="home">
+    <script type="text/javascript" src="./main.js"></script>
+    <header>
+      <a href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <a href="#apple" class="nav__link is-active">Apple</a>
+          <a href="#samsung" class="nav__link">Samsung</a>
```

Comment:
please add empty line between siblings
```suggestion
          <a href="#apple" class="nav__link is-active">Apple</a>
          
          <a href="#samsung" class="nav__link">Samsung</a>
```